packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2030,101 +2030,109 @@ mixin WidgetInspectorService {
         'result': result,
       });
     }
+
+    Map<String, Object>? additionalCallback(DiagnosticsNode node, InspectorSerializationDelegate delegate) {
+      final Object? value = node.value;
+      final RenderObject? renderObject = value is Element ? value.renderObject : null;
+      if (renderObject == null) {
+        return null;
```

Comment:
in the previous state of this callback, we were returning an empty map for the additional json in this case. Is there a reason we switched to null?

Reply:
In this case `null` and `const Map<String, Object>{}` has the same meaning I think: the caller will set the map to `const Map<String, Object>{}` if the return value is null: 
https://github.com/flutter/flutter/blob/c38731ea48b80faefc3eadcf05796e683a3ad7e1/packages/flutter/lib/src/widgets/widget_inspector.dart#L3602-L3604

Reply:
reverted to return an empty map.